doc/source/index.rst

```diff
@@ -1,67 +1,67 @@
 .. currentmodule:: nomenclature
 
-**nomenclature**: Working with IAMC-style scenario data
-=======================================================
+**nomenclature**: Working with IAMC-format project templates
+===========================================================
 
 Release v\ |version|.
 
-|license| |python| |black| |pytest| |rtd|
+|license| |doi| |python| |black| |pytest| |rtd|
 
 .. |license| image:: https://img.shields.io/badge/License-Apache%202.0-black
    :target: https://github.com/IAMconsortium/nomenclature/blob/main/LICENSE
 
-.. |black| image:: https://img.shields.io/badge/code%20style-black-000000.svg
-   :target: https://github.com/psf/black
+.. |doi| image:: https://zenodo.org/badge/375724610.svg
+   :target: https://zenodo.org/badge/latestdoi/375724610
 
 .. |python| image:: https://img.shields.io/badge/python-_3.8_|_3.9_|_3.10-blue?logo=python&logoColor=white
    :target: https://github.com/IAMconsortium/nomenclature
 
+.. |black| image:: https://img.shields.io/badge/code%20style-black-000000.svg
+   :target: https://github.com/psf/black
+
 .. |pytest| image:: https://github.com/IAMconsortium/nomenclature/actions/workflows/pytest.yml/badge.svg
    :target: https://github.com/IAMconsortium/nomenclature/actions/workflows/pytest.yml
 
 .. |rtd| image:: https://readthedocs.org/projects/nomenclature-iamc/badge/?version=latest
    :target: https://nomenclature-iamc.readthedocs.io
 
 Overview
-========
-
-The nomenclature package facilitates working with "codelists" that follow the format
-developed by the `Integrated Assessment Modeling Consortium (IAMC)
-<https://www.iamconsortium.org>`_. Codelists are yaml file based lists of allowed values
-(or codes) for dimensions of IAMC-style data, for example *regions* and *variables*.
-Using these codelists, nomenclature performs data validation to check if a provided data
-set conforms to the values in the code lists. 
-
-Additionally, it can execute "region processing", which consists of renaming of "native
-regions" and/or aggregation to "common regions" used in a project.
+--------
 
-Those two tasks are carried out by two classes:
+The **nomenclature** package facilitates validation and processing of scenario data
+for model comparison projects and scenario analysis. It allows to manage
```

Comment:
As before, I'm not sure "manage" gives the right idea to the users, maybe "work with" would be better.